bundle/src/main/java/com/adobe/acs/commons/util/ModeUtil.java

```diff
@@ -0,0 +1,187 @@
+/*
+ * #%L
+ * ACS AEM Commons Bundle
+ * %%
+ * Copyright (C) 2013 - 2015 Adobe
+ * %%
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * #L%
+ */
+package com.adobe.acs.commons.util;
+
+import java.util.HashSet;
+import java.util.Set;
+
+import org.apache.felix.scr.annotations.Activate;
+import org.apache.felix.scr.annotations.Component;
+import org.apache.felix.scr.annotations.Reference;
+import org.apache.sling.api.SlingHttpServletRequest;
+import org.apache.sling.settings.SlingSettingsService;
+import org.osgi.service.cm.ConfigurationException;
+import org.osgi.service.component.ComponentContext;
+
+import com.day.cq.commons.Externalizer;
+import com.day.cq.wcm.api.AuthoringUIMode;
+import com.day.cq.wcm.api.WCMMode;
+
+import aQute.bnd.annotation.ProviderType;
+
+//@formatter:off
+@ProviderType
+@Component(
+        immediate = true, 
+        enabled = true, 
+        label = "ACS AEM Commons - Mode Utilities", 
```

Comment:
Oh yeh - the new convention is to only have labels/descriptions when metatype=true - also enabled=true is default, so dont need to explicitly set